scripts/ci/verify_linter.py

```diff
@@ -0,0 +1,187 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import json
+from subprocess import check_output, check_call
+from pkg_resources import parse_version
+
+
+def separator_line():
+    print('-' * 100)
+
+
+class ModifiedFilesNotAllowedError(Exception):
+    """
+    Exception raise for the scenario that modified files is conflict against publish requirement.
+    Scenario 1: if modified files contain only src/index.json, don't raise
+    Scenario 2: if modified files contain not only extension code but also src/index.json, raise.
+    Scenario 3: if modified files don't contain src/index.json, don't raise.
+    """
+
+    def __str__(self):
+        msg = """
+        ---------------------------------------------------------------------------------------------------------
+        You have modified both source code and src/index.json!
+
+        There is a release pipeline will help you to build, upload and publish your extension.
+        The precondition is to put your source code inside repo Azure/azure-cli-extensions.
+
+        If you want us to help to build, upload and publish your extension, src/index.json must not be modified.
+        If your source code is outside of this repo, you have to build, upload and publish by yourself.
```

Comment:
If we identified changes in both source code and src/index.json, does it mean the source code is already in Azure/azure-cli-extensions? We don't need to state the precondition and `If your source code is outside of this repo` part?

Reply:
Updated.